### PR TITLE
ARF-75686 rails generate module accredited_representatives

### PR DESCRIPTION
**Pull request by nihil2501**
## Summary
Scaffold backend for the Accredited Representatives Portal in vets-api [ARF-75686](https://app.zenhub.com/workspaces/accredited-representative-facing-team-65453a97a9cc36069a2ad1d6/issues/gh/department-of-veterans-affairs/va.gov-team/75686)
- Feature flags: No feature flagging since no new vectors have been opened yet.
- Team: @department-of-veterans-affairs/benefits-accredited-rep-facing

## Testing done
Spec suite scaffolding, nothing more.

## What areas of the site does it impact?
This will be the backend for the Accredited Representatives Portal which is being built in `vets-website`.

## Acceptance criteria
- [x]  Scaffold passes CI

### CODE REVIEW

File: modules/accredited_representatives/Gemfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# development dependencies will be added by default to the :development group.
+gemspec
+
+# Declare any dependencies that are still in development here instead of in
+# your gemspec. These might include edge Rails or gems from your path or
+# Git. Remember to move these dependencies to your gemspec before releasing
+# your gem to rubygems.org.
+
+# To use a debugger
+# gem 'byebug', group: [:development, :test]
```
[the remark this answers is not in the document]
Comment (nihil2501) [reply modules/accredited_representatives/Gemfile:16]: Maybe we can use Ruby's overhauled debugger? Not sure if our Ruby has the `pry-byebug`-like experience that this boasts: https://github.com/ruby/irb#debugging-with-irb

**Comment by nihil2501** [comment]
> LGTM! A few questions!
> 
> Do we want to add an application_controller yet? Examples in these two patches [1](https://github.com/department-of-veterans-affairs/vets-api/commit/600c414949147ad941ba402b16e26e5b3e2e6370), [2](https://github.com/department-of-veterans-affairs/vets-api/commit/00507d03b9e64b2484a4c4f39836e2ae7ed3c78b)

We can get that in a future ticket I think, because then there will be something to exercise (in a test for instance). Possibly in the ticket that integrates the `SignIn::Authentication` module? Or maybe one of the endpoint tickets.

File: modules/accredited_representatives/README.rdoc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+= AccreditedRepresentatives
+TODO: Short description and motivation.
```
[the remark this answers is not in the document]
Comment (nihil2501) [reply modules/accredited_representatives/README.rdoc:2]: Yeah, we can add a little bit here.

File: modules/accredited_representatives/lib/accredited_representatives.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'accredited_representatives/engine'
+
+module AccreditedRepresentatives
+  # Your code goes here...
```
[the remark this answers is not in the document]
Comment (nihil2501) [reply modules/accredited_representatives/lib/accredited_representatives.rb:6]: This reminds me of similar comment directives in Rails generators. In general I actually find such directives useful in the future. In this case I am almost totally ambivalent on this line of code.

**Review by rjohnson2011 — DISMISSED** [review]

**Review by rjohnson2011 — APPROVED** [review]

**Pull request merged**